Python/pywarpx/fields.py

```diff
@@ -77,8 +76,8 @@ class _MultiFABWrapper(object):
          Note that when True, the first n-ghost negative indices will refer to the lower
          ghost cells.
     """
-    def __init__(self, mf_name, level, include_ghosts=False):
-        self.mf_name = mf_name
+    def __init__(self, mf, level, include_ghosts=False):
```

Comment:
This is fine as you have it, but I will often do this differently, where the init has both arguments, `mf` and `mf_name` and does the appropriate thing depending on which is set (instead of defining a separate class).

Reply:
I thought about that but I wasn't sure how to nicely do it while supporting the current setup where the `multifab` is newly grabbed from WarpX:
```
    @property
    def mf(self):
        # Always fetch this anew in case the C++ MultiFab is recreated
        ....
```
In this way we wouldn't be able to set `self.mf` in the `__init__` function of `_MultiFABWrapper`, but would instead need to add a new flag - something like `self.grab_mf_from_warpx = (mf is None)` and then in the `mf` property definition check if that flag is true for whether to return a fetched multifab or the one given at initialization. I thought the current approach is somewhat cleaner than that. What do you think? Do you have a nicer way in mind?